src/prototypo-canvas.js

```diff
@@ -202,6 +210,7 @@ function drawTypographicFrame() {
 	this.typographicFrame.xHeight.size.height = 1 / this.zoom;
 	this.typographicFrame.capHeight.size.height = 1 / this.zoom;
 	this.typographicFrame.linearDraggingHelper ? this.typographicFrame.linearDraggingHelper.strokeWidth = 2 / this.zoom : null;
+	this.typographicFrame.snappingHelper ? this.typographicFrame.snappingHelper.strokeWidth = 2 / this.zoom : null;
 	this.typographicFrame.linearDraggingHelper ? this.typographicFrame.linearDraggingHelper.dashArray = [ 8 / this.zoom, 8 / this.zoom ] : null;
```

Comment:
Wouldn't it work though?
According to specs
variablename = (condition) ? value1:value2 
That what I had in mind when writing this code, maybe I'm wrong!